scss/_patterns_pull-quotes.scss

```diff
@@ -4,26 +4,67 @@
   color: $color-mid-dark;
   display: inline-block;
   position: absolute;
+  width: 0.5em;
 }
 
 @mixin vf-pull-quote {
   border: 0;
   margin: $spv-outer--scaleable 0 $spv-outer--scaleable + $sp-unit;
   overflow: visible;
   position: relative;
+
+  &.has-image {
+    margin-top: calc(#{$spv-inner--x-large} + #{$spv-outer--scaleable});
+  }
 }
 
 // Pull quote styling
 @mixin vf-p-pull-quotes {
-  .p-pull-quote {
-    padding: 0 $sph-outer * 2;
+  .p-pull-quote,
+  .p-pull-quote--small,
+  .p-pull-quote--large {
+    @include vf-pull-quote;
+
+    // this needs to be nested to increase the specificity
+    .p-pull-quote__citation {
```

Comment:
All element class should be nested within the block in BEM. We don't want styling to effect anything if you remove the block class from the container.

Reply:
That's really agains BEM rules of having single class selectors.

The `.p-pull-quote__` part makes sure this class name is part of the pattern and shouldn't be used outside.
If we nest the selectors, why don't just make them `.p-pull-quote .citation`?